docs: add standalone game support page (#240)
RetroAchievements/docs: docs/general/standalone-support.md

---
title: Standalone Support

> description: Describes how games can receive standalone support, as well as the conditions- and requirements.

Reviewer: ```suggestion
description: Guide to RetroAchievements support for standalone games, including requirements, evaluation criteria, and the proposal process for games that don't use emulators.
```

The `description` field is what immediately appears when someone finds the page using a search engine. This content here:

![Screenshot 2025-05-05 at 7 50 13 PM](https://github.com/user-attachments/assets/ef1b0085-b3ab-4be3-9052-12aa7dee8358)

How do you feel about this suggestion? It might help the page make more sense if someone is about to land on it sideways from Google (I suspect that's how this page will most often be found).

---

# Standalone Games

We offer support for Standalone games, which do not use an emulator to connect to RetroAchievements. This is done by directly making HTTP calls to our emulator integration API (the "Connect API"), be it through directly modifying the game or creating a mod that has the code required to connect to the RetroAchievements server. However, all Standalone support must abide strict requirements, as well as approval by the RetroAchievements Admins. Even if all the guidelines below are met, this will not guarantee we will accept your proposal. For the best chances of success, it is vital that you present a highly-detailed plan to the admins.

[[toc]]

## What Are the Requirements to Work on a Standalone

> Standalone support cannot be proposed by just about anyone, as there are strict non-negotiable requirements that must be met.

Reviewer: Stronger phrasing:
```suggestion
Standalone support for a game requires strict non-negotiable requirements to be met.
```

> - The proposer must either be a Developer on RetroAchievements, or a Junior Developer with at least one published set. As these achievement sets still have to follow our global [Achievement Design](https://docs.retroachievements.org/developer-docs/achievement-design.html) rules. This is not something we can trust anyone with who has not showed us their proficiency in set design before.

Reviewer: Minor: fixed a sentence fragment.
```suggestion
- The proposer must either be a Developer on RetroAchievements, or a Junior Developer with at least one published set, as these achievement sets still have to follow our global [Achievement Design](https://docs.retroachievements.org/developer-docs/achievement-design.html) rules. This is not something we can trust anyone with who has not showed us their proficiency in set design before.
```

   - Alternatively, the proposer may suggest a standalone without having developed a set, but must do so with the support of a RetroAchievements Developer who will be in charge of set design.

> - You must be able to enforce Hardcore restrictions as shown in the [Global Leaderboard and Achievement Hunting Rules](https://docs.retroachievements.org/guidelines/users/global-leaderboard-and-achievement-hunting-rules.html). Such examples include black/whitelisting other mods that reduce the challenge of achievements, or disallowing multiplayer to prevent unfair advantages. These advantages should only be reserved for Softcore players, if applicable.

Reviewer: It may also be good to mention disabling/blocking in-game cheats in this area, as it's probable the standalone developer has direct access to this capability.

> - We expect you to have knowledge of the tools required to add Standalone support to a game. This varies depending on the game you are looking at, with some delving into code languages such as C#, while others have extensive modding tools available.

Reviewer: ```suggestion
- We expect you to have knowledge of the tools required to add Standalone support to a game. This varies depending on the game you are working on, with some games requiring expertise with programming languages such as C#, while other games may require extensive modding tools available which can be leveraged for developing an achievement set.
```

### What Do We Look for in Standalone Support

> We will be listing some guidelines below as for what we are looking for. However, do keep in mind that at all times, the Admin team of RetroAchievements makes the decision as to whether we want to proceed with your proposal. A good plan- or prototype must therefore always be presented.

Reviewer: Added some bold here for emphasis.
```suggestion
We will be listing some guidelines below as for what we are looking for. However, do keep in mind that at all times, **the Admin team of RetroAchievements makes the decision as to whether we want to proceed with your proposal.** A good plan- or prototype must therefore always be presented.
```
